zipkin-server/README.md

```diff
@@ -370,6 +370,20 @@ Example usage:
 $ RABBIT_ADDRESSES=localhost java -jar zipkin.jar
 ```
 
+### gRPC Collector (Experimental)
+
+You can enable a gRPC span collector endpoint by setting `GRPC_SERVER_ENABLED=true`. The
+`zipkin.proto3.SpanService/Report` endpoint will run on the same port as normal http (9411).
+
+
+If you need to deploy a collector-only server, you can disable other http endpoints like so:
+
+```bash
+GRPC_COLLECTOR_ENABLED=true HTTP_COLLECTOR_ENABLED=false QUERY_ENABLED=false java -jar zipkin.jar
```

Comment:
ps I'm just now noticing our env convention is backwards.

it might be ok to keep it, to be in parity with http. or we can add COLLECTOR_HTTP_ENABLED as a fallback instead of propagating the backwards convention to grpc